docs/contributing/get-started-developing.md

```diff
@@ -0,0 +1,153 @@
+
+```bash
+# Tag the image
+docker tag \
+  "fluxcd/helm-operator:$(./docker/image-tag)" \
+  "localhost:5000/helm-operator:$(./docker/image-tag)"
+
+# Push the image
+docker push \
+  "localhost:5000/helm-operator:$(./docker/image-tag)"
+```
+
+## Deploying your changes
+
+You can now deploy your changes to the Kind cluster using your newly built and pushed image:
+
+```bash
+# Create fluxcd namespace
+kubectl create namespace fluxcd
+
+# Apply HelmRelease CRD
+kubectl apply -f deploy/crds.yaml
+
+# Install helm-operator using pushed image
+helm upgrade -i helm-operator fluxcd/helm-operator \
+  --namespace fluxcd \
+  --set helm.versions=v3 \
+  --set image.repository=localhost:5000/helm-operator \
+  --set image.tag="$(./docker/image-tag)"
+```
+
+You should now have an instance of the Flux Helm Operator running with your changes!
+
+## See your changes in action
+
+To see your changes in action, first deploy a HelmRelease setting the new `echoMessage` field:
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: podinfo
+  namespace: default
+spec:
+  echoMessage: This is a test
+  releaseName: podinfo
+  chart:
+    repository: https://stefanprodan.github.io/podinfo
+    version: 2.1.0
+    name: podinfo
+  values:
+    replicaCount: 1
+EOF
+```
+
+Once the release is deployed you can view the helm-operator pods logs to see your emitted message:
+
+```bash
+> kubectl logs --selector=app=helm-operator
+
+...
+ts=2020-04-15T20:35:16.249414781Z caller=release.go:78 component=release release=podinfo targetNamespace=default resource=default:helmrelease/podinfo helmVersion=v3 info="This is a test"
+ts=2020-04-15T20:35:16.540354042Z caller=release.go:272 component=release release=podinfo targetNamespace=default resource=default:helmrelease/podinfo helmVersion=v3 info="running installation" phase=install
+ts=2020-04-15T20:35:16.89689355Z caller=helm.go:69 component=helm version=v3 info="creating 3 resource(s)" targetNamespace=default release=podinfo
+ts=2020-04-15T20:35:16.938861204Z caller=release.go:281 component=release release=podinfo targetNamespace=default resource=default:helmrelease/podinfo helmVersion=v3 info="installation succeeded" revision=2.1.0 phase=install
+...
+```
+
+## Congratulations!
+
+You did it! you made your first change to the HelmRelease CRD, built, deployed and ran an instance of the Flux Helm Operator to see it in action.
+
+You should now hopefully be more comfortable with making changes and running the Flux Helm Operator locally, and be ready to tackle your [first issue](https://github.com/fluxcd/helm-operator/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22).
+
+To find out more about the Flux Helm Operator community and our contribution workflow have a look at the [contributing guide](https://docs.fluxcd.io/projects/helm-operator/en/stable/contributing/introduction/).
```

Comment:
```suggestion
To find out more about the Flux Helm Operator community and our contribution workflow have a look at the [contributing guide](introduction.md).
```

`mkdocs` figures out relative links to other documents by itself and transforms them to the appropriate URLs :-)